src/test/java/model/input/file/XlsxFileTest.java

```diff
@@ -59,7 +53,7 @@ public void testRead() throws Exception {
 
 		assertEquals(new StringValue("string"), row.getValue(table.getColumn("string")));
 		assertEquals(new FloatValue(0.666f), row.getValue(table.getColumn("float")));
-		assertEquals(new IntValue(666), row.getValue(table.getColumn("int")));	
+		assertEquals(new IntValue(666), row.getValue(table.getColumn("int")));
```

Comment:
Je hebt in deze test dezelfde code staan als in XlsFileTest, kan je dat niet samenvoegen?